projects/example/project.yaml

```diff
@@ -0,0 +1,4 @@
+# Provide the home page for your project:
+homepage: "https://github.com/google/oss-fuzz"
+# Provide the e-mail for the primary contact:
+primary_contact: "kcc@google.com"
```

Comment:
Yes, but after CF files the bug. I want to have a real auto-reported bug as part of the docs.